api/src/test/resources/org/openmrs/module/mksreports/include/outpatientConsultationTestDataset.xml

```diff
@@ -62,4 +62,10 @@
 		 value_numeric="[NULL]" value_datetime="[NULL]" value_drug="[NULL]"
 		 uuid="91fc8bff-0c2a-4b4f-b8fe-13abecdee795" />
 
+	<obs obs_id="4006" person_id="7" concept_id="21" encounter_id="3"
+		 obs_datetime="2008-09-02 00:00:00.0" location_id="1" value_coded="1923"
+		 comments="" creator="1" date_created="2008-08-18 14:09:35.0" voided="false"
+		 value_coded_name_id="[NULL]" value_complex="[NULL]" value_text="[NULL]"
+		 value_numeric="[NULL]" value_datetime="[NULL]" value_drug="[NULL]"
+		 uuid="cf613b43-0e89-4c8d-9821-92bff858b087" />
```

Comment:
If we want to demonstrate that the report can provide totales across genders, we need to attach obs to two patients of different genders. Person ID'd 7 is a female, so can you also record another diagnosis for person ID'd 6 (who is a male)?

Reply:
The existing obs took care of that, that's why I didn't add.

Reply:
There was an existing obs with Person ID'd 6

Reply:
Hmm. I responded without looking at the code at the code I wrote. There are 4 obs in the file. 3 for Person with ID 6(which is male) for each of the diagnosis and 1 obs for Person with ID 7(female) for the malaria diagnosis. So this resulted in a total of two people with Malaria, one male and one female

Reply:
Ok that's perfect then.

Reply:
OK that's perfect then.